### PR TITLE
Add missing CSV::generate_line libdef

### PR DESCRIPTION
**Pull request by deecewan**
See https://docs.ruby-lang.org/en/2.6.0/CSV.html#method-c-generate_line

### Motivation

There was a place in my codebase that used this method, and I noticed it wasn't typed. Added the type.

### Test plan

I didn't add tests for this - made the change directly on GitHub. Can add a test usage if you like, but it's probably not necessary

### CODE REVIEW

File: rbi/stdlib/csv.rbi [hunk 1]
```diff
@@ -369,6 +369,20 @@ class CSV < Object
...
+  #
+  # The `options` parameter can be anything [`CSV::new()`](https://docs.ruby-lang.org/en/2.6.0/CSV.html#method-c-new) understands.
+  # This method understands an additional `:encoding` parameter to set the base
+  # Encoding for the output.  This method will try to guess your Encoding from
+  # the first non-`nil` field in `row`, if possible, but you may need to use
+  # this parameter as a backup plan.
+  #
+  # The `:row_sep` `option` defaults to `$INPUT_RECORD_SEPARATOR`
+  # (`$/`) when calling this method.
+  sig { params(row: T::Array[String], options: T.untyped)}
```
Comment (connorshea) [inline rbi/stdlib/csv.rbi]:
```suggestion
  sig { params(row: T::Array[String], options: T.untyped) }
```

File: rbi/stdlib/csv.rbi [hunk 2]
```diff
@@ -369,6 +369,20 @@ class CSV < Object
...
+  #
+  # The `options` parameter can be anything [`CSV::new()`](https://docs.ruby-lang.org/en/2.6.0/CSV.html#method-c-new) understands.
+  # This method understands an additional `:encoding` parameter to set the base
+  # Encoding for the output.  This method will try to guess your Encoding from
+  # the first non-`nil` field in `row`, if possible, but you may need to use
+  # this parameter as a backup plan.
+  #
+  # The `:row_sep` `option` defaults to `$INPUT_RECORD_SEPARATOR`
+  # (`$/`) when calling this method.
+  sig { params(row: T::Array[String], options: T.untyped) }
```
Comment (elliottt) [inline rbi/stdlib/csv.rbi]: This sig is missing a `returns`.

Comment (deecewan) [reply rbi/stdlib/csv.rbi]: sorry about that @elliottt - fixed :)

**Comment by elliottt** [comment]
Successful build on the stripe monorepo: http://go/54UDCe0Q

**Pull request merged**